devtools-ci-index.yaml

```diff
@@ -2156,7 +2167,7 @@
             ci_cmd: '/bin/bash cico_build_deploy.sh'
             saas_git: saas-launchpad
             prj_name: launchpad-preview
-        - '{ci_project}-{git_repo}':
```

Comment:
this job will need to be removed manually from cico when the merge goes through

Reply:
It should have the same job name afaik